server/broadcast.go

```diff
@@ -434,6 +444,13 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string)
 		if dlErr != nil {
 			return dlErr
 		}
+
+		if verifier != nil {
+			if err := verify(verifier, cxn, sess, seg, res); err != nil {
```

Comment:
If we end up initializing a [verifier-less policy](https://github.com/livepeer/go-livepeer/pull/1096/files#diff-205b550ef0ac99b368da16cfb883960bR664) responsible for sig & pixel count checking when running in on-chain mode, then pixel count checking would become synchronous instead of asynchronous. That might be an ok tradeoff for cleaner separation of verification related concerns - just noting this for now.